lib/grapevine_web/live/post_live.ex

```diff
@@ -8,12 +8,14 @@ defmodule GrapevineWeb.PostLive do
   def mount(_params, %{"user_token" => token}, socket) do
     posts = Posts.show_all()
     categories = Posts.get_categories()
+    categories = [%{id: 0, name: "All"} ] ++ categories
```

Comment:
```suggestion
    categories = [%{id: 0, name: "All"} | categories]
```

https://elixirschool.com/en/lessons/basics/collections/#lists-0

Reply:
So, both will have the same result, but in IMHO using `++` to concatenation these lists seems to provide more visual and readability, even when someone is new in Elixir or be learning use `++` instead of `[item | []]`.